src/cpu/rv64/intrin_rvv.hpp

```diff
@@ -0,0 +1,16 @@
+// Referenced from
```

Comment:
Copyright banner?

Reply:
Sorry for the late reply! Could you plz tell me how to resolve the copyright issues?